### PR TITLE
Add recipe for moments

### PR DESCRIPTION
**Pull request by apragsdale**
Starting the recipe for moments (bitbucket.org/simongravel/moments). Needs a bit of work, I'm sure - pinging @lecorguille, who suggested to get this started. Thanks!!

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by molpopgen** [comment]
This one is failing for reasons that seem unrelated to building, etc., the package.  Needs some assistance from someone who knows the guts.

File: recipes/moments/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{% set name="moments" %}
+{% set version="1.0.5" %}
+
+package:
+  name: {{ name }}
+  version: '{{ version }}'
```
Comment (molpopgen) [inline recipes/moments/meta.yaml:6]: @apragsdale -- this line could be the problem?  In other recipes, I don't see the single quotes around the version.

File: recipes/moments/build.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+cd $SRC_DIR/moments
```
Comment (lecorguille) [inline recipes/moments/build.sh:3]: When Conda download the src archive, it also `cd`in it.
So you can consider in the build.sh that you are already in the src directory.
You can remove this line.

**Review by lecorguille — CHANGES_REQUESTED** [review]

**Comment by apragsdale** [comment]
Thanks @lecorguille, that looks to have done it.

Should I rebase/squash at this point, or anything else that needs to be done?

**Review by molpopgen — APPROVED** [review]

**Review by molpopgen — APPROVED** [review]

**Comment by molpopgen** [comment]
@lecorguille -- you may need to "Approve" the changes?  Merging is currently blocked and we can't figure out why?

**Review by lecorguille — APPROVED** [review]

**Comment by lecorguille** [comment]
Congrats @apragsdale !

**Pull request merged**